src/views/SlideDetailView.vue

```diff
@@ -57,13 +57,15 @@ const handleSave = async () => {
   }
 }
 const handleDelete = async () => {
-  try {
-    await deleteSlideDetail(id)
-    toast.success('スライドを削除しました')
-    router.push('/')
-  } catch (e) {
-    if (e instanceof Error) {
-      toast.error(`エラーが発生しました: ${e.message}`)
+  if (window.confirm('本当にこのスライドを削除しますか？')) {
+    try {
+      await deleteSlideDetail(id)
+      toast.success('スライドを削除しました')
+      router.push('/')
+    } catch (e) {
+      if (e instanceof Error) {
+        toast.error(`エラーが発生しました: ${e.message}`)
+      }
```

Comment:
早期returnするとネストが深くならなくてよさそうです
こんな感じ
```suggestion
  if (!window.confirm('本当にこのスライドを削除しますか？')) {
  	return
  }
	try {
	await deleteSlideDetail(id)
	toast.success('スライドを削除しました')
	router.push('/')
	} catch (e) {
	if (e instanceof Error) {
	  toast.error(`エラーが発生しました: ${e.message}`)
	}
```